### DISCUSSION TITLE
How to disable minecraft item tags showing up when hitting "U"

### DISCUSSION THREAD

**Discussion by Evansch0** [shedaniel/RoughlyEnoughItems#971] [Q & A]
exactly as the title states, game lags heavily in larger modpacks when going to the tabs tag. 1.18.2 fabric.

**Comment by drav0011** [comment]
This feature is useful for mod/modpack developers and maybe for more advanced modpack players, but for average users it just a thing on the screen which they cannot understand.
Ideally there would be a toggle in the client settings to show or hide. And I would hide it by default

Related to https://github.com/shedaniel/RoughlyEnoughItems/issues/512

**Reply by shedaniel** [reply]
The tag lag should be fixed now but I don't feel to comfortable hiding the tag category for now, a way to easily hide and reorder categories will come in the future.

**Reply by Lancet-crow** [reply]
Can I do it through code, maybe? My tag lag spikes just killing my game performance. Config can't do it, as I see.

**Reply by shedaniel** [reply]
You can do it via KubeJS
